### PR TITLE
Update ruby and rails

### PR DESCRIPTION
**Pull request by matthewbentley**
I think I got all the places that need updating

### CODE REVIEW

**Comment by ajm188** [comment]
If it passes, :ship:.

TODO:
- [ ] ruby 2.2 -> 2.3
- [ ] rails 4 -> 5 (maybe)

**Comment by matthewbentley** [comment]
Isn't rails 5 still beta?

**Comment by ajm188** [comment]
Yeah, but it's on its third beta :stuck_out_tongue:

**Comment by ajm188** [comment]
hmmm, try changing `rake <task>` to `bundle exec rake <task>` ?

File: Gemfile.lock [hunk 1]
```diff
@@ -342,3 +340,6 @@ DEPENDENCIES
   uglifier (>= 1.3.0)
   watir-webdriver
   web-console (~> 2.0)
+
+BUNDLED WITH
```
Comment (ajm188) [inline Gemfile.lock]: ooh actually can we remove this? it makes minor version conflicts in different people's bundlers a barrier to entry

Comment (ajm188) [inline Gemfile.lock]: I understand why bundler added this, but I also hate it

**Comment by ajm188** [comment]
Every single failure has to do with methods defined in "shoulda-matchers". Looking at Gemfile.lock, it had a major version bump. We should either pin it to `~> 2.0`, or figure out what the upgrade process is

**Comment by matthewbentley** [comment]
They all passed :) +1s welcome

**Comment by ajm188** [comment]
:+1: (even though I have a commit on here)

**Comment by ajm188** [comment]
Also, I opened an issue for the shoulda business.

**Pull request merged**